### PR TITLE
Add/Update Cmdlet Docs

### PR DESCRIPTION
**Pull request by saleens7**

### CODE REVIEW

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+Module Name: MicrosoftTeams
+title: Set-CsTeamsCustomBannerText
+author: wblocker
+ms.author:
```
Comment (get-itips) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:5]: please fill this metadata

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## SYNOPSIS
+
+Enables administrators to configure a custom text on the banner displayed when compliance recording bots start recording the call. 
+
+## SYNTAX
+
+### Identity (Default)
+```
+Set-CsTeamsCustomBannerText [[-Id] <Guid>] [<CommonParameters>]
```
Comment (get-itips) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:20]: This is missing Text and Description parameters

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+## DESCRIPTION
+
+Updates a single instance of custom banner text.
+
+## EXAMPLES
+
+### Example 1
+PS C:\> Set-CsTeamsCustomBannerText -Identity CustomText
```
Comment (get-itips) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:30]: It seems the example is missing -Text parameter?

File: teams/teams-ps/teams/Remove-CsTeamsCustomBannerText.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+Module Name: MicrosoftTeams
+title: Remove-CsTeamsCustomBannerText
+author: wblocker
+ms.author:
```
Comment (get-itips) [inline teams/teams-ps/teams/Remove-CsTeamsCustomBannerText.md:5]: please fill this metadata

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## SYNOPSIS
+
+Enables administrators to configure a custom text on the banner displayed when compliance recording bots start recording the call. 
+
+## SYNTAX
+
+### Identity (Default)
+```
+New-CsTeamsCustomBannerText [[-Id] <Guid>] [<CommonParameters>]
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText.md:20]: it seems this is missing Text and Description parameters

**Comment by get-itips** [comment]
@saleens7 author is your github username and ms.author is your microsoft alias

File: skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md [hunk 6]
```diff
@@ -100,6 +100,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -CustomBanner
+Reference to the Custom Banner text in the storage
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md:104]:
```suggestion
References the Custom Banner text in the storage.

```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 7]
```diff
@@ -104,6 +104,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -CopilotWithoutTranscript
+
+This policy give admins the ability to enable or disable copilot use without having to start transcript.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:109]:
```suggestion
This policy gives admins the capability to enable or disable copilot use without having to start transcript.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 8]
```diff
@@ -104,6 +104,38 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CopyRestriction
+
+Enable this setting to allow users to disable right-click or Ctrl+C to copy, Copy link, Forward message, and Share to Outlook for meeting chat messages
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:125]:
```suggestion
Enable this setting to allow users to disable right-click or Ctrl+C to copy, Copy link, Forward message, and Share to Outlook for meeting chat messages.
```

File: skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md [hunk 9]
```diff
@@ -127,6 +127,22 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### CustomBanner
+Reference to the Custom Banner text in the storage
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md:131]:
```suggestion
References the Custom Banner text in the storage.

```

File: skype/skype-ps/skype/Set-CsTeamsMeetingConfiguration.md [hunk 10]
```diff
@@ -354,6 +354,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LimitPresenterRolePermissions
+When set to True, users within the Tenant will have their presenter role capabilities limited.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingConfiguration.md:358]:
```suggestion
When set to True, users within the organization will have their presenter role capabilities limited.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 11]
```diff
@@ -130,7 +130,39 @@ Position: Named
...
+Required: False
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CopyRestriction
+
+Enable this setting to allow users to disable right-click or Ctrl+C to copy, Copy link, Forward message, and Share to Outlook for meeting chat messages
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:153]:
```suggestion
Enable this setting to allow users to disable right-click or Ctrl+C to copy, Copy link, Forward message, and Share to Outlook for meeting chat messages.
```

File: teams/teams-ps/teams/Get-CsTeamsCustomBannerText.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsCustomBannerText
+```
+
+Gets the properties of all instances of the TeamsCustomBannerText.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsCustomBannerText.md:36]:
```suggestion
This example gets the properties of all instances of the TeamsCustomBannerText.
```

File: teams/teams-ps/teams/Get-CsTeamsCustomBannerText.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+### Example 2
+```powershell
+PS C:\> Get-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Gets the properties of the CustomText instance of TeamsCustomBannerText.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsCustomBannerText.md:43]:
```suggestion
This example gets the properties of the CustomText instance of TeamsCustomBannerText.
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+PS C:\> New-CsTeamsCustomBannerText -Id 123e4567-e89b-12d3-a456-426614174000 -Description "Custom Banner Text Example" -Text "Custom Text"
+```
+
+Creates an instance of TeamsCustomBannerText with the name CustomText.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText.md:35]:
```suggestion
This example creates an instance of TeamsCustomBannerText with the name CustomText.
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText.md [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+
+## PARAMETERS
+
+### -Id
+The Identity of the CustomBannerText. You do not need to provide an Id as the backend will generate it for you. However if you wish to provide your own ID, you can provide your own GUID. Please note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText.md:40]:
```suggestion
The Identity of the CustomBannerText. You do not need to provide an ID as the backend will generate it for you. However, if you wish to provide your own ID, you can provide your own GUID. Note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText.md [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+Applicable: Microsoft Teams
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Text
+The text that the tenant admin would like to set in policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText.md:55]:
```suggestion
The text that you would like to set in the policy.
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+```yaml
+Type: String
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
+The description that the tenant admin would like to set to identify what this text represents.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText.md:66]:
```suggestion
The description that you would like to set to identify what this text represents.
```

File: teams/teams-ps/teams/Remove-CsTeamsCustomBannerText.md [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+
+Removes a single instance of custom banner text.
+
+## EXAMPLES
+
+### Example 1
+PS C:\> Remove-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Removes a TeamsCustomBannerText instance with the name "CustomText".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsCustomBannerText.md:33]:
```suggestion
This example removes a TeamsCustomBannerText instance with the name "CustomText".
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+
+Updates a single instance of custom banner text.
+
+## EXAMPLES
+
+### Example 1
+PS C:\> Set-CsTeamsCustomBannerText -Id 123e4567-e89b-12d3-a456-426614174000 -Description "Custom Banner Text Example" -Text "Custom Text"
+```
+
+Sets the properties of the CustomText instance of TeamsCustomBannerText.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:33]:
```suggestion
This example sets the properties of the CustomText instance of TeamsCustomBannerText.
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+Applicable: Microsoft Teams
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Text
+The text that the tenant admin would like to set in policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:53]:
```suggestion
The text that you would like to set in the policy.
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+```yaml
+Type: String
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
+The description that the tenant admin would like to set to identify what this text represents.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText.md:64]:
```suggestion
The description that you would like to set to identify what this text represents.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @saleens7, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@saleens7 Gentle reminder.

**Pull request closed without merging**